1055 building block event studies for causal inference

VirginiaMirabile opened the pull request:
Hi all, 

here is the pull request for the building block on event studies. 
I focused on the application in the case of staggered treatment design. 

Some of the key aspects are: 

- Why event studies? What are the advantages with respect to other causal inference methods. 
- Introducing the context of staggered treatment designs and why event studies are appropriate to use. 
- Application in R, with an example dataset from a field experiment in Tilburg. 

Just a small reminder, I still need to upload the data so the code cannot be run yet if you don't have the dataset. 

Thank you, let me know if any changes are needed.

VirginiaMirabile commented:
Update: the link to access the dataset has been included in a code snippet.

lachlandeer commented:
@srosh2000 -- can you look this over soon?

Thanks!

srosh2000 commented on content/topics/Analyze/causal-inference/panel-data/event_studies.md:
```diff
@@ -0,0 +1,263 @@
+<img src = "../images/event_timeplot.png" width="500">
+</p>
+
+
```
You may want to add a disclaimer that here you assume treatment effects to be homogenous across time and groups. The estimated event time coeffs are based on comparisons of newly treated groups (routes) relative to routes that have not yet been treated as well as comparisons of newly treated groups relative to _already treated groups._ If at all treatment effects are heterogenous, comparing newly treated with already treated (so called "forbidden comparisons") can lead to bias in the treatment effect. One would then use the Bacon decomposition method proposed by Goodman-Bacon, 2021 to perform a diagnosis as to whether this is the case. This is an important step before going into a robust methods to overcome the bias. Given that we dont have a BB on bacon decomposition yet I think it would be nice if you could add it in this one. If you find that there is no bias then you can conclude that your specification is robust to this.

srosh2000 replied:
Here's the link to the paper by Andrew Bacon: Goodman-Bacon, A. (2021). Difference-in-differences with variation in treatment timing. Journal of econometrics, 225(2), 254-277.

srosh2000 replied:
`bacondecomp` package can be used in R

srosh2000 replied:
@lachlandeer @shrabasteebanerjee Let me know what you think about adding bacon decomposition part to this BB?

lachlandeer replied:
i'd be cautious about "adding too much" to one topic. It gets overwhelming or gets explained in too much of a hurry to be clear to a reader.

srosh2000 reviewed (changes requested):
@lachlandeer @shrabasteebanerjee Let me know what you think about adding bacon decomposition part to this BB?

lachlandeer commented:
hey @VirginiaMirabile -- have you addressed these comments from @srosh2000

VirginiaMirabile commented:
> hey @VirginiaMirabile -- have you addressed these comments from @srosh2000

Hi @lachlandeer, I will address them once I submit my thesis in a few days. I let @srosh2000 know in a recent causal inference meeting we had.

VirginiaMirabile commented:
Hi @srosh2000, I have implemented your feedback and made some slight adjustments to the text. Regarding the images, since the website has been changed they cannot be previewed locally, I have checked the code for them and everything should be working fine once the bb is merged. 

Let me know if there's anything I can do!

shrabasteebanerjee commented:
Learnt from @valerievossen than this is ready so merging!
